employee-onboarding-A2A-reference-implementation/setup_wso2_is.py

```
@@ -0,0 +1,1073 @@
+
+    def create_oidc_application(
+        self,
+        name: str,
+        description: str,
+        grant_types: list[str],
+        callback_urls: list[str] = None,
+        template_id: Optional[str] = None,
+    ) -> dict:
+        existing = self.find_application(name)
+        if existing:
+            warn(f"Application '{name}' already exists – fetching full details")
```

Ruff (0.15.7)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)